wiki/Game_modifier/Easy/en.md

```diff
@@ -42,7 +42,7 @@ In [osu!](wiki/Game_Modes/osu!), the Easy mod decreases [circle size (CS)](/wiki
 
 ![EZ gameplay osu!](img/EZ-comparison-osu.jpg "Comparison between a regular play (left) vs a play with the Easy mod enabled (right) in osu!")
 
-Although, it is worth mentioning that many players do not find the use of the Easy mod to be helpful in decreasing the relative difficulty of beatmaps, especially high-intensity ones. The argument for this goes that the decreased approach rate creates a messy, difficulty-to-read situation where the actual speed of the beatmap is not very well reflected visually (displayed below).
+Although, it is worth mentioning that many players do not find the use of the Easy mod to be helpful in decreasing the relative difficulty of beatmaps, especially high-intensity ones. The argument for this goes that the decreased approach rate creates a messy, difficult-to-read situation where the actual speed of the beatmap is not very well reflected visually (displayed below).
```

Comment:
```suggestion
Although, it is worth mentioning that many players do not find the use of the Easy mod to be helpful in decreasing the relative difficulty of beatmaps, especially high-intensity ones. The argument for this goes that the decreased approach rate creates a messy, difficult-to-read situation where the actual speed of the beatmap is not very well reflected visually (displayed above).
```

Reply:
wonder which things are displayed below here.
thought if this changed, others translations have to get changes as well.